src/pages/publication/FilesAndLicensePanel.tsx

```diff
@@ -1,25 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
 import Box from '../../components/Box';
 import TabPanel from '../../components/TabPanel/TabPanel';
-import UppyFileUpload from './files_and_license_tab/UppyFileUpload';
+import FileUploader from './files_and_license_tab/FileUploader';
+import FileCard from './files_and_license_tab/FileCard';
 
 interface FilesAndLicensePanelProps {
   goToNextTab: (event: React.MouseEvent<any>) => void;
 }
 
 const FilesAndLicensePanel: React.FC<FilesAndLicensePanelProps> = ({ goToNextTab }) => {
+  const [files, setFiles] = useState<any[]>([]);
```

Comment:
Venter med å lage type for File til neste PR, da det vil skje en del endringer med den der uansett